release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -263,6 +263,12 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 
 In {product-title} 4.8, the `rhcos4-moderate` profile is now complete. The `ocp4-moderate` profile will be completed in a future release.
 
+[discrete]
+[id="ocp-4-8-preparation-HAProxy-2.2"]
+=== Ingress Controller upgraded to HAProxy 2.2
+
+The {product-title} Ingress Controller is upgraded to HAProxy version 2.2.
```

Comment:
Do we want to indicate the micro version (i.e., 2.2.13) in the detail, leaving the title to just 2.2?

Reply:
I can add the micro version here.